settings/src/components/totp.js

```diff
@@ -213,7 +213,7 @@ function SetupForm( { handleEnable, verifyCode, setVerifyCode, qrCodeUrl, secret
 				required={ true }
 			/>
 
-			<div>
+			<div class="wporg-2fa__verify-code wporg-2fa__button-wrapper">
```

Comment:
Is the `width` from `wporg-2fa__verify-code` necessary on the `div`? I'd assume that the label text inside the button would set its width automatically. Hardcoding it could be a problem in the future if we start to translate the text.

Is there a semantic connection between the name/purpose of that class and this button? They don't seem to be related to me, but I could be missing it.

Since we're overriding the `margin-bottom` anyway, I wonder if it'd be better to  remove `wporg-2fa__verify-code` from this element? We could add a `width` to `wporg-2fa__button-wrapper` if there's a need for it.

Reply:
No worries at all, [we're all bad at it](https://martinfowler.com/bliki/TwoHardThings.html)!

Reply:
Classic 😆